### PR TITLE
Fixes #1140 - Temporarily removed Safari Add-on link

### LINKED ISSUE #1140: Remove Safari Add-on link
Reported @ https://github.com/webcompat/web-bugs/issues/2913

### PR DESCRIPTION
**Pull request by moraveyo**
r? @miketaylr

### CODE REVIEW

File: webcompat/templates/addon-links.html [hunk 1]
```diff
@@ -42,7 +42,7 @@
     <span class="wc-Navbar-linkDownload-label-word">Download our </span> Opera Extension
  </span>
 </a>
-{%- elif browser == 'safari' -%}
+{%- elif browser == 'safari WHEN IT IS READY' -%}
```
Comment (miketaylr) [inline webcompat/templates/addon-links.html]: This is clever. :)

Even thought it achieves what we want, I think were better off if we're not shipping "dead" code. 

Can you remove the entire `elif` clause, and the addons directory as well? When we're ready to add it back, its not hard to dig up what we had from the git history.

Thanks!

**Comment by miketaylr** [comment]
Looks good -- thanks!

(btw, feel free to re-ping in a review once you've made updates to your PR)

**Comment by miketaylr** [comment]
I'm gonna clean up the history before merging -- when I do a rebase from a fork, I generally do something like:

`git pull --rebase upstream master`

This avoids the extra "merge branch blah" commit.

**Comment by miketaylr** [comment]
Hm, it's not working, which might mean you haven't given me permission to change your PR branches. :)

If you'd like to clean up the history yourself, can you try the following:

1. make sure you have an upstream remote (this points back to the original project that you forked from).

`git remote -v`

If you see something like this, your'e good:

```
(ignoring origin)
upstream	git@github.com:webcompat/webcompat.com.git (fetch)
upstream	git@github.com:webcompat/webcompat.com.git (push)
```

If not, you can add it like so:

`git remote add origin https://github.com/webcompat/webcompat.com.git`

Now, fetch the latest upstream:

`git fetch upstream`

And rebase your branch from its master (assuming you're on this current branch)

`git pull --rebase upstream master`

Now we just need to squash the 2 commits into one.

You can do an interactive rebase of the last 2 commits like so:

`git rebase -i HEAD~2`

You should see something like this:

screen shot 2017-04-14 at 11 02 57 am

Change the second commit from `keep` to `f`, which means it will merge its changes with the previous commit, and use the first commit's commit message.

Once that's done, you can push back to this branch via:

`git push -f` (assuming you had set un upstream branch URL, it might complain at you and give you the commit to do this)

Once that's done (phew!), ping me and we'll be ready to merge.

**Comment by moraveyo** [comment]
r? @miketaylr
That git magic!.. Hope I've done it right.
The last command was `git push -f --set-upstream origin issues/1140/1`.

How can I  give you permission to change my PR branches?
The "Allow edits from maintainers." checkbox is on.  Isn't that it?

**Comment by miketaylr** [comment]
> The "Allow edits from maintainers." checkbox is on. Isn't that it?

Ah yeah, that's it. So it just means I goofed something up (...which is the normal state of affairs).

**Pull request merged**